packages/@lwc/rollup-plugin/src/index.js

```diff
@@ -56,6 +58,11 @@ module.exports = function rollupLwcCompiler(pluginOptions = {}) {
         },
 
         resolveId(importee, importer) {
+            if (!semver.satisfies(this.meta.rollupVersion, pkg.peerDependencies.rollup)) {
```

Comment:
This check will run every time rollup resolves a file which is not ideal. I can either be moved to the `buildStart` or the `options` rollup hook to only run once per build.

Reply:
Thanks, moved it to `buildStart` as that seems like the most logical place to put it.